curriculum/challenges/english/blocks/zh-a1-learn-certification-introduction/691ecaaed19d057a2870c77b.md

```diff
@@ -0,0 +1,71 @@
+---
+id: 691ecaaed19d057a2870c77b
+title: How to Set Up the Chinese Keyboard on Your Computer
+challengeType: 24
+dashedName: how-to-set-up-the-chinese-keyboard-on-your-computer
+lang: zh-CN
+---
+
+# --description--
+
+Chinese is spoken in many regional varieties. In this curriculum, you will learn **Mandarin**, the most widely used standard form of spoken Chinese.
+
+Chinese is written in two major forms: Simplified and Traditional Chinese. This curriculum will focus on **Simplified Chinese**.
+
+Since Chinese characters are not phonetic, a system called **Pinyin**​ is used to represent their sounds. Pinyin is a romanization system that uses the Latin alphabet to indicate how Chinese characters are pronounced. It serves as an essential tool for beginners to learn the pronunciation of the language and to type Chinese on digital devices.
+
+To input Chinese, you type the Pinyin, and a bar of candidate characters will appear. From there, you can slide or tap to choose the one you want.
+
+If you are using your computer for the Chinese curriculum, you'll need to make sure it's ready to type Pinyin.
```

Comment:
Similarly, we either should change the wording or remove this lesson, as setting up the keyboard isn't required.